src/content/en/_redirects.yaml

```diff
@@ -974,4 +974,64 @@ redirects:
   to: https://developer.chrome.com/blog/a-new-experimental-feature-style-scoped
 
 - from: /web/updates/2012/03/CSS-layout-gets-smarter-with-calc
-  to: https://developer.chrome.com/blog/css-layout-gets-smarter-with-calc
\ No newline at end of file
+  to: https://developer.chrome.com/blog/css-layout-gets-smarter-with-calc
+
+- from: /web/updates/2012/03/Optimizing-JavaScript
+  to: https://developer.chrome.com/blog/optimizing-javascript
+
+- from: /web/updates/2012/04/Big-boost-to-DOM-performance-WebKit-s-innerHTML-is-240-faster
+  to: https://developer.chrome.com/blog/big-boost-to-dom-performance-webkit-s-innerhtml-is-240-faster
+
+- from: /web/updates/2012/04/datalist-landed-in-Chrome-Canary
+  to: https://developer.chrome.com/blog/datalist-landed-in-chrome-canary
+
+- from: /updates/2012/04/Processing-XHR2-file-uploads-in-PHP
```

Comment:
Missed the `/web/` prefix here. I've fixed in another PR. Running `npm run test` prior to doing the push should catch these errors.

cc: @rachelandrew